core/templates/dev/head/services/AssetsBackendApiService.ts

```diff
@@ -91,10 +91,26 @@ oppia.factory('AssetsBackendApiService', [
                          window.MozBlobBuilder ||
                          window.MSBlobBuilder;
           if (exception.name === 'TypeError' && window.BlobBuilder) {
-            var blobBuilder = new BlobBuilder();
-            blobBuilder.append(data);
-            assetBlob = blobBuilder.getBlob(assetType.concat('/*'));
+            try {
+              var blobBuilder = new BlobBuilder();
```

Comment:
I don't think the issue introduced because of constructing BlobBuilder. (As the error log says `Failed to construct 'Blob':  The 2nd argument is not of type Object` i.e, unable to run `new Blob(list, undefined (or !Object))`).

But I'm fine with both logs.